src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java

```diff
@@ -138,12 +153,24 @@ private WebElement findSpeakerIcon() {
   }
 
   private void waitFor(final Predicate<AutoplayVuap> predicate, final long timeout) {
-    new FluentWait<AutoplayVuap>(this)
+    new FluentWait<>(this)
         .withTimeout(timeout, TimeUnit.SECONDS)
         .pollingEvery(1, TimeUnit.SECONDS)
         .until(predicate);
   }
 
+  private boolean isOverlayVisible() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "isOverlayVisible" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)